x-pack/platform/packages/shared/kbn-ai-assistant/src/hooks/__storybook_mocks__/use_knowledge_base.ts

```diff
@@ -5,17 +5,20 @@
  * 2.0.
  */
 
+import { KnowledgeBaseState } from '@kbn/observability-ai-assistant-plugin/common';
 import { UseKnowledgeBaseResult } from '../use_knowledge_base';
 
 export function useKnowledgeBase(): UseKnowledgeBaseResult {
   return {
-    install: async () => {},
-    isInstalling: false,
+    isLoading: false,
```

Comment:
why is this isLoading now? It seems less descriptive than isInstalling, given we are polling and waiting for the install process to finish, and not just waiting for a request to finish "loading" something.